### DISCUSSION TITLE
Parts appear outside of viewport of librecad initially

### DISCUSSION THREAD

**Discussion by possibilities** [mozman/ezdxf#572] [Q&A] [answered]
I'm exporting a model using ezdxf but when I open it in LibreCad everything is outside of the viewport until I zoom out or scroll. Is there some way to affect this with ezdxf? Thanks!

**Comment by mozman** [comment, marked as answer]
Please verify this or any issue with a different CAD application than LibreCAD. LibreCAD is not a reliable DXF renderer.

**Comment by possibilities** [comment]
Yup looks fine elsewhere, thank you!
